src/entities/conditional-logic/model/conditions.ts

```diff
@@ -74,3 +78,226 @@ export const doesNotIncludeValue = <TItem>(
+  const selectedTime = getTimeBasedOnFieldName(fieldName, timeRange);
+  return selectedTime
+    ? timeToMinutes(selectedTime) < timeToMinutes(time)
+    : false;
+};
+
+export const isEqualToTimeRange = (
+  timeRange: { startTime: HourMinute | null; endTime: HourMinute | null },
+  { time, fieldName }: { time: HourMinute; fieldName: string },
+): boolean => {
+  const selectedTime = getTimeBasedOnFieldName(fieldName, timeRange);
+  return selectedTime
+    ? timeToMinutes(selectedTime) == timeToMinutes(time)
+    : false;
+};
+
+export const isNotEqualToTimeRange = (
+  timeRange: { startTime: HourMinute | null; endTime: HourMinute | null },
+  { time, fieldName }: { time: HourMinute; fieldName: string },
+): boolean => {
+  const selectedTime = getTimeBasedOnFieldName(fieldName, timeRange);
+  return selectedTime
+    ? timeToMinutes(selectedTime) !== timeToMinutes(time)
+    : false;
+};
+
+export const isBetweenTimesRange = (
+  timeRange: { startTime: HourMinute | null; endTime: HourMinute | null },
+  {
+    minTime,
+    maxTime,
+    fieldName,
+  }: { minTime: HourMinute; maxTime: HourMinute; fieldName: string },
+): boolean => {
+  const selectedTime = getTimeBasedOnFieldName(fieldName, timeRange);
+  const selectedTimeInMinutes = selectedTime
+    ? timeToMinutes(selectedTime)
+    : null;
+
+  return (
+    selectedTimeInMinutes !== null &&
+    selectedTimeInMinutes >= timeToMinutes(minTime) &&
+    selectedTimeInMinutes <= timeToMinutes(maxTime)
+  );
+};
+
+export const isOutsideOfTimesRange = (
+  timeRange: { startTime: HourMinute | null; endTime: HourMinute | null },
+  {
+    minTime,
+    maxTime,
+    fieldName,
+  }: { minTime: HourMinute; maxTime: HourMinute; fieldName: string },
+): boolean => {
+  const selectedTime = getTimeBasedOnFieldName(fieldName, timeRange);
+  const selectedTimeInMinutes = selectedTime
+    ? timeToMinutes(selectedTime)
+    : null;
+
+  return (
+    selectedTimeInMinutes !== null &&
+    (selectedTimeInMinutes < timeToMinutes(minTime) ||
+      selectedTimeInMinutes > timeToMinutes(maxTime))
+  );
+};
+
+export const isGreaterThanTime = (
+  input: { hours: number; minutes: number },
+  time: { hours: number; minutes: number },
+): boolean => {
+  const inputMinutes = timeToMinutes(input);
+  const conditionMinutes = timeToMinutes(time);
+
+  const result = inputMinutes > conditionMinutes;
+
+  return result;
+};
+
+export const isLessThanTime = (
+  input: { hours: number; minutes: number },
+  time: { hours: number; minutes: number },
+) => {
+  const inputMinutes = timeToMinutes(input);
+  const conditionMinutes = timeToMinutes(time);
+
+  const result = inputMinutes < conditionMinutes;
+
+  return result;
+};
+
+export const isEqualToTime = (
+  input: { hours: number; minutes: number },
+  time: { hours: number; minutes: number },
+) => {
+  const inputMinutes = timeToMinutes(input);
+  const conditionMinutes = timeToMinutes(time);
+
+  const result = inputMinutes == conditionMinutes;
```

Comment:
Ditto. These will always be numbers, so we can validate with `===`.
```suggestion
  const result = inputMinutes === conditionMinutes;
```